integrationExamples/gpt/pbjs_example_gpt.html

```diff
@@ -39,7 +38,7 @@
         var d = document;
         var pbs = d.createElement("script");
         pbs.type = "text/javascript";
-        pbs.src = '/build/dist/prebid.js';
+        pbs.src = '/build/dev/prebid.js';

```

Comment:
Should integration examples include the dev build, or is it better to keep the dist reference if people are likely to copy-paste this code?